test/Microsoft.AspNet.Razor.Test/CodeGenerators/CSharpTagHelperRenderingTest.cs

```diff
@@ -848,32 +848,36 @@ public static TheoryData DesignTimeTagHelperTestData
                         DefaultPAndInputTagHelperDescriptors,
                         new List<LineMapping>
                         {
-                            BuildLineMapping(documentAbsoluteIndex: 14,
-                                             documentLineIndex: 0,
-                                             generatedAbsoluteIndex: 493,
-                                             generatedLineIndex: 15,
-                                             characterOffsetIndex: 14,
-                                             contentLength: 11),
-                            BuildLineMapping(documentAbsoluteIndex: 62,
-                                             documentLineIndex: 3,
-                                             documentCharacterOffsetIndex: 26,
-                                             generatedAbsoluteIndex: 1289,
-                                             generatedLineIndex: 39,
-                                             generatedCharacterOffsetIndex: 28,
-                                             contentLength: 0),
-                            BuildLineMapping(documentAbsoluteIndex: 122,
-                                             documentLineIndex: 5,
-                                             generatedAbsoluteIndex: 1634,
-                                             generatedLineIndex: 48,
-                                             characterOffsetIndex: 30,
-                                             contentLength: 0),
-                            BuildLineMapping(documentAbsoluteIndex: 88,
-                                             documentLineIndex: 4,
-                                             documentCharacterOffsetIndex: 12,
-                                             generatedAbsoluteIndex: 1789,
-                                             generatedLineIndex: 54,
-                                             generatedCharacterOffsetIndex: 19,
-                                             contentLength: 0)
+                            BuildLineMapping(
```

Comment:
Did these change at all or did you just format it? Want to make sure I'm not missing something.